script.js

```diff
@@ -0,0 +1,15 @@
+// test Run A/C
+let desiredTemp = 74;
+// let actualTemp = 78;
+//test run heat 
+// let actualTemp = 70;
+//test standby
+let actualTemp = 70;
+
+if (actualTemp > desiredTemp) {
+    console.log('Run A/C');
+}   else if (actualTemp < desiredTemp) {
+    console.log('Run heat');
+}   else (actualTemp === desiredTemp); {
```

Comment:
The semicolon after (actualTemp === desiredTemp) is messing this up a bit. I'm getting 2 logs if both temps are not equal. 

Ending 'else' doesn't need the statement in ( ). It is like a default statement that will run if none of the other if statements are triggered. Should look like:
} else {
  console.log('Standy");
}